### ISSUE TITLE
create an index files inside of examples folder and connect all of our examples/schemas there

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/sd#516]
**Is your feature request related to a problem? Please describe.**
I want to splify our work with next step - we can reduce logic of connecting schemas and examples by having one file that connect all of them.
later we'll create a method, that will return both schema and example into our test file.
it maybe not make our code better, but will help us later

**Is it was created as task at GitHub issues? Please provide an issue number or link**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by atherdon** [comment]
Ok, let's also add and export from this file a method, that will return to us one files, when it's necessary. something like get('allergy') and it will return our variable

**Comment by elnur004** [comment]
what do you recommend? how to do this?

**Comment by atherdon** [comment]
the easiest way is to combine all of our variables into array.
so diets, and other will be inside of array. aka ```[ 'diets':diets,
'allergy':allergy ]```...

then inside of that function we'll use that variable as array key in order
to return only one file with data, not all of them. show me some examples
at PR and i'll guide you forward

>

**Comment by elnur004** [comment]
@atherdon need we add a function in index.js file or play.js one for return the file with data?

**Comment by elnur004** [comment]
btw sorry for the late response. i've been busy lately

**Comment by atherdon** [comment]
yes, index that will be a separated point where we connect to files. and no
problem at all - we don't have rush - so take your time

>

**Comment by elnur004** [comment]
@atherdon @vadim9999 is there any task for me?

**Comment by atherdon** [comment]
of course! @elnur004 
did you remember our initial task, related to creating constants? I think now you can crack it without any problems. i move that task into fetch plugin, if you want - i'll assign it to you

**Comment by elnur004** [comment]
@atherdon yeah i'm waiting!

**Comment by atherdon** [comment]
@elnur004 grab this one: https://github.com/GroceriStar/groceristar-fetch/issues/448#issuecomment-507060281

Am i rihgt - i can close this task?

**Comment by elnur004** [comment]
@atherdon if everything has been done, you can
